app/views/hyrax/base/_show_actions.html.erb

```diff
@@ -0,0 +1,65 @@
+<%# OVERRIDE Hyrax 3.5.0 to remove feature work button %>
+
+<div class="row show-actions button-row-top-two-column">
+  <div class="col-sm-6">
+    <% if !workflow_restriction?(presenter) %>
+      <% if presenter.show_deposit_for?(collections: @user_collections) %>
+        <input type="checkbox"  style="display:none" name="batch_document_ids[]" id="batch_document_<%= presenter.id %>" value="<%= presenter.id %>" class="batch_document_selector" checked="checked" />
+        <%= button_tag t('hyrax.dashboard.my.action.add_to_collection'),
+                      class: 'btn btn-default submits-batches submits-batches-add',
+                      data: { toggle: "modal", target: "#collection-list-container" } %>
+      <% end %>
+      <%# OVERRIDE begin %>
+      <% if presenter.work_featurable? && current_account.name != 'dc' %>
```

Comment:
Would you want to repurpose the constant from AccountsHelper?  that magic `'dc'` might haunt us.